### DISCUSSION TITLE
MiniScribe 20 successfully imaged with Initiator Mode

### DISCUSSION THREAD

**Discussion by JosBarranquero** [BlueSCSI/BlueSCSI-v2#207] [General]
Wasn't really sure where to report this, but I was successful in imaging a MiniScribe 20 (yeah, I got one that still works!) with my BlueSCSI.
I'll attach the log file, with the debug flag set.
[MiniScribe_imaging_log.txt](https://github.com/user-attachments/files/17606638/MiniScribe_imaging_log.txt)

**Comment by erichelgeson** [comment]
Thanks for the report! I've added it to the list of working drives: https://github.com/BlueSCSI/BlueSCSI-v2/wiki/Initiator-Mode#compatibility

**Comment by hdunsany** [comment]
I'm dropping in here to (hopefully!) save someone else a bit of frustration and/or confusion: I'm trying to replicate this process and image an old Apple MiniScribe 20SC pulled from a Mac SE. I've followed the Initiator Mode guide to the letter but the BlueSCSI would always get stuck waiting for the MiniScribe. On a whim I just tried enabling the "BACK_FEED" jumper on my BlueSCSI v2 and suddenly it's able to image the drive. It's getting a ton of errors in the imaging process (I'm watching via serial on my MacBook Pro) but at least it's progress!
